.travis.yml

```diff
@@ -1,9 +1,17 @@
-language: java
-jdk:
-  - oraclejdk8
-addons:
-  sonarcloud:
-    organization: "improbable-keanu"
-script:
-  - ./gradlew check
-  - 'if [[ ! -z $SONAR_TOKEN ]]; then sonar-scanner; fi'
+matrix:
+  include:
+    - language: java
+      jdk: oraclejdk8
+      addons:
+        sonarcloud:
+          organization: "improbable-keanu"
+      script:
+        - ./gradlew clean build -x :keanu-python:build -i
```

Comment:
Why do we need to run `keanu-python:build` here - aren't we running this again in the python section of the build?

Reply:
The `-x` means exclude